### ISSUE TITLE
Tweedie loss function

### ISSUE THREAD

**Issue by friedsela** [microsoft/LightGBM#3266]
It seems to me that even if the prediction equals the true value, the loss (as defined) is not 0. Is this so? Why does this make sense? Is there a reference for this?

**Comment by guolinke** [comment]
refer to the discussion in https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/150614

**Comment by friedsela** [comment]
I am aware of this discussion on Kaggle. Maybe I am missing something but I don't see how this answers my questions. It seems to me that if what I am saying is right, then the loss function does not force the predictions to be equal the target, but something else and I find it weird, therefore I am asking for a reference, and I didn't see any in the discussion you referred me to.

**Comment by guolinke** [comment]
Tweedie is not invented by us, and I didn't use it before, so I cannot provide you the exact reason.
But I think for a loss function, it is not necessary that should be zero when output equals to target.

**Comment by friedsela** [comment]
Ok, Thank you.

**Comment by Zhylkaaa** [comment]
@guolinke hello, you mentioned that it wasn't invented by light gbm authors, can you point to the reference implementation? Or maybe @lorentzenchr can help with that, sorry for taking your time, but I cant find the answer for how this loss is implemented(not only me #5088 ).
Thank you!
